starter/src/app/home/home.component.spec.ts

```diff
@@ -54,8 +53,7 @@ describe(`Home`, () => {
 						provide: STARK_HTTP_SERVICE,
 						useValue: jasmine.createSpyObj("StarkHttpService", ["executeSingleItemRequest", "executeCollectionRequest"])
 					},
-					{ provide: STARK_LOGGING_SERVICE, useValue: jasmine.createSpyObj("StarkLoggingService", ["debug", "error"]) },
-					{ provide: STARK_SESSION_SERVICE, useValue: jasmine.createSpyObj("StarkSessionService", ["login"]) }
+					{ provide: STARK_LOGGING_SERVICE, useValue: jasmine.createSpyObj("StarkLoggingService", ["debug", "error"]) }
```

Comment:
Could we use MockStarkLoggingService here instead of creating a new jamine spyObj ? 😊

Reply:
I've created this issue to track this refactoring: #367